API_usage.md

```diff
@@ -5,14 +5,14 @@ Scientist publish the data to this STAC catalog to make it available to the user
 
 Follow the guide below to publish datasets to the VEDA STAC catalog.
 
-## Prepare the data
+## STEP I: Prepare the data
 
 VEDA supports inclusion of cloud optimized GeoTIFFs (COGs) to its data store.
 
 ### Creating COGs
 
 1. Make sure the projection system is embedded in the COG
-2. Make sure the there's an associated `NoData` value in the COG
+2. Make sure the there's an associated `NoData` value in the COG. Make sure it's not a huge number, -9999 is usually a good number.
```

Comment:
```suggestion
2. Make sure the COG has a defined `NoData` value already **before you create the overviews** (cloud-optimize the file). Otherwise, overviews and on-the-fly reprojections will be polluted with the fill value. Good `NoData` values for floats are `NaN` and for integers are the maximum or minimum possible value of the data type.
```